sources/reference/api/experiment.md

```diff
@@ -115,6 +116,13 @@ information.
 
 [ext]: #extensions
 
+[Contribution][contrib] describes valuable properties of the target system,
+such as "reliability: or "scalability", that an experiment contribute to. This
```

Comment:
Change "scalability' to "durability", and "an experiment contributes to" - small typo.